catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/faceting/FacetValueCount.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.operation.faceting;
+
+public class FacetValueCount {
+    private String value;
+    private long count;
+
+    FacetValueCount(String value, long count) {
```

Comment:
The incoming value is the minimum count required in order for the faceting information to be reported, which is different from the outgoing value - the number of results reported.
An `int` seems appropriate.